convert2rhel/unit_tests/utils_test.py

```diff
@@ -214,6 +215,89 @@ def test_is_rpm_based_os(self):
         assert is_rpm_based_os() in (True, False)
 
 
+class TestDownloadPkg(object):
+    def test_download_pkg_failed_download_overridden(self, monkeypatch, caplog):
+
+        monkeypatch.setattr(system_info, "releasever", "7server")
+        monkeypatch.setattr(system_info, "version", namedtuple("Version", ["major", "minor"])(7, 0))
```

Comment:
You could replace this with 

```suggestion
    @centos7
    def test_download_pkg_failed_download_overridden(self, monkeypatch, caplog):
```

that decorator will initialize and mock SystemInfo for you, of course, you maybe need to import it with: 

```python
from convert2rhel.unit_tests import conftest
```

Reply:
Yeah thank you for this suggestion, I saw that decorator with some of the other tests but I wasn't 100% sure how to use it. So I just stuck with using monkeypatch, I will get that changed and push it!